db/migrate/20250225000050_remove_contents_from_issue_contents.rb

```diff
@@ -0,0 +1,95 @@
+# For auto load
+FullTextSearch::Migration
+
+class RemoveContentsFromIssueContents < ActiveRecord::Migration[5.2]
```

Comment:
What version of Redmine did you use to generate this migration file?

It seems that `[5.2]` is too old.